pipeline/terraform/main.tf

```diff
@@ -71,26 +74,28 @@ module "catalogue_pipeline_2021-01-22" {
   # reindexer topic names.
 
   sierra_adapter_topic_arns = [
-    local.sierra_reindexer_topic_arn,
+    /*local.sierra_reindexer_topic_arn,*/
     local.sierra_merged_bibs_topic_arn,
     local.sierra_merged_items_topic_arn,
   ]
 
   miro_adapter_topic_arns = [
-    local.miro_reindexer_topic_arn,
+    /*local.miro_reindexer_topic_arn,*/
     local.miro_updates_topic_arn,
   ]
 
   mets_adapter_topic_arns = [
-    local.mets_reindexer_topic_arn,
+    /*local.mets_reindexer_topic_arn,*/
     local.mets_adapter_topic_arn,
   ]
 
   calm_adapter_topic_arns = [
-    local.calm_reindexer_topic_arn,
+    /*local.calm_reindexer_topic_arn,*/
     local.calm_adapter_topic_arn,
   ]
 
+  extra_rds_instances = 0
```

Comment:
Why not just `rds_instances`? You could even add a custom validation rule to make sure it's always >=1, `extra` to me is a bit confusing as it isn't clear what I'm augmenting

Reply:
Mmm, let me think about this some more.

The "extra" is meant to be "in addition to the base instance, which is always running". Maybe calling it `rds_instances` would make more sense, but I'm also tempted by Robert's idea of a "reindexing" module.